lascli/actions/workflows.py

```diff
@@ -0,0 +1,241 @@
+import json
+import os
+import requests
+import uuid
+import collections
+import traceback
+
+from datetime import datetime
+from las import Client
+from typing import Optional
+
+from ..util import wrap_output, capture_return
+
+
+# FIXME: Get this from somewhere else.
+COMPONENT_ASSEST_URI = 'https://raw.githubusercontent.com/LucidtechAI/las-docs/master/tutorials/simple-demo/backend/src/Invoice/assets/jsRemoteComponent.js'  
+
+created_ids = collections.defaultdict(list)
+ 
+
+def create_workflow_spec(
+    model_id: str, preprocess_transition_id: str, manual_transition_id: str,
+    postprocess_transition_id: str, tag: str 
+):
+    return {
+        'definition': {
+            'Comment': f'Auto-generated workflow for model {model_id}. {tag}.',
+            'StartAt': 'Preprocess',
+            'States': {
+                'Preprocess': {
+                    'Type': 'Task',
+                    'Resource': preprocess_transition_id,
+                    'Next': 'CheckValidation'
+                },
+                'CheckValidation': {
+                    'Type': 'Choice',
+                    'Choices': [{
+                        'Variable': '$.needsValidation',
+                        'BooleanEquals': False,
+                        'Next': 'Postprocess'
+                    }, {
+                        'Variable': '$.needsValidation',
+                        'BooleanEquals': True,
+                        'Next': 'Validate'
+                    }]
+                },
+                'Validate': {
+                    'Type': 'Task',
+                    'Resource': manual_transition_id,
+                    'Next': 'Postprocess'
+                },
+                'Postprocess': {
+                    'Type': 'Task',
+                    'Resource': postprocess_transition_id,
+                    'End': True
+                }
+            }
+        }
+    }
+    
+
+def create_field_config(las_client: Client, model_id: str):
+    model = las_client.get_model(model_id)
+    field_config = model['fieldConfig']
```

Comment:
Not all models (older ones) have fieldConfig, but maybe that's okay. Should we print a better error message if that's the case or ignore it since it will not be happening very often?